source/model/Proposal.ts

```diff
@@ -57,15 +57,22 @@ export class ProposalModel {
     const { body } = await service.get<Proposal[]>(
       'dataset/proposals.min.json'
     );
-    const [finished, processing] = body.reduce(
-      ([finished, processing], proposal) => {
-        if (proposal.stage === 4) finished.push(proposal);
-        else processing.push(proposal);
+    const [finished, processing] = body
+      .map(({ tags, authors, champions, ...rest }) => ({
+        tags,
```

Comment:
```suggestion
        tags,
```

可以吧 tags 转换成一个 `Record<string, true>`

然后就这么用

```
record.tags['ECMA-262'] && record.edition < year
```

```
record.tags['co-champion'] && record.edition < year
```

Reply:
那我不但要自己转数据结构，还要定义两套 `interface`……